src/completion.ts

```diff
@@ -70,13 +81,32 @@ export function asCompletionItem(
     if (sourceDisplay) {
         item.detail = Previewer.plainWithLinks(sourceDisplay, filePathConverter);
     }
+    const wordRange = completionContext.optionalReplacementSpan ? Range.fromTextSpan(completionContext.optionalReplacementSpan) : undefined;
```

Comment:
Let's create the range earlier, where context is created, to avoid creating range for every completion.